src/lib/es2015.iterable.d.ts

```diff
@@ -144,7 +144,7 @@ interface MapConstructor {
 interface WeakMap<K extends WeakKey, V> {}
 
 interface WeakMapConstructor {
-    new <K extends WeakKey, V>(iterable: Iterable<readonly [K, V]>): WeakMap<K, V>;
+    new <K extends WeakKey = WeakKey, V = any>(iterable?: Iterable<readonly [K, V]> | null): WeakMap<K, V>;

```

Comment:
Wouldn't `unknown` be a better default here? It would more closely match the previous behavior (unconstrained type param is implicitly constrained to `unknown`, at least in strict mode)
```suggestion
    new <K extends WeakKey = WeakKey, V = unknown>(iterable?: Iterable<readonly [K, V]> | null): WeakMap<K, V>;

```

Reply:
The other `WeakMap` constructor uses `any`:
<https://github.com/microsoft/TypeScript/blob/e3d234cfc8653d14e907ae84f918d5bd18546619/src/lib/es2015.collection.d.ts#L65-L68>

And making this change broke some tests.